includes/managers/controls.php

```diff
@@ -1216,6 +1216,10 @@ private function should_clean_stack( $stack ): bool {
 	}
 
 	public function add_display_conditions_controls( Controls_Stack $controls_stack ) {
+		if ( Utils::has_pro() ) {
```

Comment:
We should not add these controls if the user has Pro. If the user has Pro, then Pro will decide if the Feature or the Promotion will show. These controls here in `add_display_conditions_controls()` should only be added if the user only has the free plugin.